src/ecc/codes/toric.jl

```diff
@@ -1,4 +1,29 @@
-"""The Toric code."""
+"""The Toric code [kitaev2003fault](@cite).
+
+Illustration of a 2x2 toric code, where qubits are located on the edges:
+
+```
+|--1-(Z)-2--|
+| (X) 5     6
+|--3--|--4--|
+|     7     8
+|     |     |
+```
+
+It's important to note that the toric code has periodic boundary conditions, which means that the top and bottom sides are essentially glued together, as are the left and right sides.
```

Comment:
```suggestion
It is important to note that the toric code has periodic boundary conditions, which means that the top and bottom sides are essentially glued together, as are the left and right sides.
```

Minor style change. In modern American English, no one cares about this, but I try to stick to the older "no contractions" style.